platform/admin/core/admin-core-impl/src/main/java/org/codice/ddf/admin/core/impl/AdminConsoleService.java

```diff
@@ -334,7 +334,7 @@ public boolean updateGuestClaimsProfile(String pid, Map<String, Object> configur
       String profile = (String) profileObj;
       guestClaimsHandlerExt.setSelectedClaimsProfileName(profile);
 
```

Comment:
:question: Was the problem that GuestClaimsHandlerExt was not initialized properly before being used here?

Reply:
I assume that was the problem, though I can't be sure. Probably some timing dependency on parsing the metatype.xml file. From an engineering perspective, it makes sense to use the map that is passed into the method because that is what the user is seeing on the screen. And it is also correct. I feel like something unnecessarily complicated is going on when the method parameter has the values we need but is ignored.

Reply:
I am inclined to agree, the old way looks wrong on general principle.